### PR TITLE
feat(benchmark): adding basic benchmarks

### PR DESCRIPTION
**Pull request by jGundermann**
- [x] Add memory usage

### CODE REVIEW

File: python/tests/benchmarks/readers.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+
+def fastexcel_read(test_file_path: str):
+    reader = read_excel(test_file_path)
+    for sheet_name in reader.sheet_names:
+        try:
+            sheet = reader.load_sheet_by_name(sheet_name)
+            sheet.to_arrow()
+        except Exception:
+            pass
```
Comment (lukapeschke) [inline python/tests/benchmarks/readers.py:31]: I don't think we should try/except here

Comment (jGundermann) [reply python/tests/benchmarks/readers.py:31]: currently when we have a empty sheet we have a exception that raises. Maybe we need to fix this in another issue?

Comment (jGundermann) [reply python/tests/benchmarks/readers.py:31]: issue logged here. 

https://github.com/ToucanToco/fastexcel/issues/111

Comment (lukapeschke) [reply python/tests/benchmarks/readers.py:31]: Nice. Can you please add a `FIXME` comment here ? We should be ready to merge afterwards :slightly_smiling_face:

**Pull request merged**